src/com/missafternoon/budjen/transactions/TransactionTest.java

```diff
@@ -22,23 +25,23 @@ public void givenMoney_returnsDescriptionAndValueAndType() {
         final Transaction transaction = makeTransaction();
         assertEquals("Your account has been credited with £10.00", transaction.getSummary());
         assertEquals(1000, transaction.getMoney());
-        assertEquals(TransactionType.CREDIT, transaction.getType());
+        Assert.assertEquals(TransactionType.CREDIT, transaction.getType());
```

Comment:
Why was this `Assert.` prefix required? Looks like we're already importing `assertEquals` statically.